### PR TITLE
Updated testing-toolkit version

### PR DESCRIPTION
**Pull request by vijayg10**

### CODE REVIEW

File: ml-testing-toolkit/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description:  ml-testing-toolkit Helm chart for Kubernetes
 name: ml-testing-toolkit
-version: 11.0.0
-appVersion: "10.4.1"
+version: 11.0.1
+appVersion: "backend 11.0.1 front-end 11.0.0"
```
Comment (rmothilal) [inline ml-testing-toolkit/Chart.yaml:5]: is this suppose to be 
`mojaloop/ml-testing-toolkit: v11.0.1; mojaloop/ml-testing-toolkit-ui: v11.0.0`

**Review by rmothilal — CHANGES_REQUESTED** [review]
please see comment

File: ml-testing-toolkit/Chart.yaml (hunk 1 above)
Comment (vijayg10) [reply ml-testing-toolkit/Chart.yaml]: Thanks Rajiv, I updated accordingly.

**Review by rmothilal — APPROVED** [review]
happy

**Pull request merged**